src/uproot/reading.py

```diff
@@ -2047,7 +2047,7 @@ def key(self, where):
                 return key
             elif cycle is None and last is None:
                 last = key
-            elif cycle is None and last.fCycle < key.fCycle:
+            elif cycle is None and abs(last.fCycle) < abs(key.fCycle):
```

Comment:
```suggestion
            # Follow ROOT's behaviour in comparing negative fCycle values
            elif cycle is None and abs(last.fCycle) < abs(key.fCycle):
```

Reply:
Just adding a comment for future ease of understanding.

(yes, we can git-blame, but this is low-cost high-readability).